index.html

```diff
@@ -1,18 +1,53 @@
 <!DOCTYPE html>
 <html lang="pl">
-
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
     <title>TITLE</title>
     <link rel="stylesheet" href="styles/main.css" type="text/css" media="all" />
-</head>
+    <link rel="stylesheet" href="styles/basics.css" />
```

Comment:
Zaimportowaliśmy to do main.css, nie potrzebujemy już tego tutaj :D

Reply:
Błąd z serii "Wydawało mi się, że to zrobiłam" :D Usunięte.